en/salesforce/src/docs/ms_composer_release_notes.adoc

```diff
@@ -9,6 +9,34 @@ New versions are automatically available once they are released; you don't need
 //| Fixed an issue in NetSuite Connector in which nested fields containing the `list` type failed. | COCN-726
 //* Composer now supports the static creation of lists. You can now dynamically complete fields that are enabled for lists even if those fields do not have an existing list to map to in the flow.
 
+== May 31, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* In NetSuite Connector, empty field values are ignored by default; however, to clear empty values instead, you can now select the `Clear on Null Values` option.
```

Comment:
Suggest:
In NetSuite Connector, empty field values are ignored by default. To clear empty values, you can now select the `Clear on Null Values` option.